### DISCUSSION TITLE
the element property show in imodel different in microstation

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#5429] [Q&A]
the element show many property in microstation ,and how get the element property by Presentation or eccql?
![6660](https://user-images.githubusercontent.com/18022720/233408431-93886056-0763-480f-b603-40ebc7c2e9f3.png)
![66666](https://user-images.githubusercontent.com/18022720/233408446-df5b5a5b-85e6-4e1b-a5d1-c9fcdbde2e71.png)

**Comment by ColinKerr** [comment]
Closing as duplicate.  Please use this discussion: https://github.com/iTwin/itwinjs-core/discussions/5427
